.github/workflows/build_docs.yml

```diff
@@ -109,10 +109,9 @@ jobs:
         # TODO: Enable NVDec/NVEnc
         conda install --quiet -y 'ffmpeg>=4.1' pkg-config
         pip --quiet install cmake>=3.18.0 ninja
-        cd packaging
-        . ./pkg_helpers.bash
-        setup_build_version
-        cd ../
```

Comment:
Can we append a date if BUILD_VERSION is not set?

Without it, it's difficult when the nightly doc was deployed last.